musicbot/bot.py

```diff
@@ -662,6 +679,16 @@ def _autopause(player):
                 await self.change_presence(activity=game)
                 self.last_status = game
 
+    async def update_now_playing_topic(self, entry=None, is_paused=False):
+        for id in self.config.nowplaying_topic_channels:
+            channel = self.get_channel(id)
+            if entry and is_paused:
+                await channel.edit(topic=':pause_button: **' + entry.title + '**')
+            elif entry and not is_paused:
+                await channel.edit(topic=':arrow_forward: **' + entry.title + '**')
+            elif entry is None:
+                await channel.edit(topic=self.config.default_topic)
+
```

Comment:
Could you please add a permission check? so it doesn't just throw an exception if the bot doesn't have Manage Channels.
If it doesn't have permission, log a warning. otherwise, continue to the rest of the function.